northd/northd.c

```diff
@@ -2843,6 +2860,13 @@ join_logical_ports(const struct sbrec_port_binding_table *sbrec_pb_table,
                                  "port %s is a switch port", op->key);
                 }
             }
+        } else if (op->nbsp && !lsp_is_router(op->nbsp)) {
```

Comment:
Same here.  I don't think we can just ignore and remove this "else if".

@dceara Wdyt ?

Reply:
@numansiddique  I guess you meant "I think we can just ignore and remove.." right?

Reply:
Yes I meant the same.  Sorry for the typo